packages/mobile/src/identity/reducer.ts

```diff
@@ -437,9 +617,16 @@ export const isVerificationStateExpiredSelector = (state: RootState) => {
   )
 }
 
-export const isRevealAllowed = ({ identity: { lastRevealAttempt } }: RootState) => {
-  return (
-    !lastRevealAttempt ||
-    timeDeltaInSeconds(Date.now(), lastRevealAttempt) > ATTESTATION_REVEAL_TIMEOUT_SECONDS
-  )
+export const isFeelessVerificationStateExpiredSelector = (state: RootState) => {
+  const { lastFetch } = state.identity.feelessVerificationState
+  return !lastFetch || timeDeltaInSeconds(Date.now(), lastFetch) > VERIFICATION_STATE_EXPIRY_SECONDS
+}
+
+// Use this only as part of the notification display logic
+export const tryFeelessOnboardingSelector = ({
```

Comment:
Can you clarify this a bit more? Why is it unsuitable elsewhere?

Reply:
That's an old comment I forgot to delete, thanks for flagging. We can use this elsewhere.